src/server/pkg/testutil/random.go

```diff
@@ -0,0 +1,19 @@
+package testutil
+
+import (
+	"fmt"
+	"math/rand"
+	"strconv"
+	"time"
+)
+
+// SeedRand seeds the rand package with a seed based on the time and returns
+// a string with the seed embedded in it.
+func SeedRand(customSeed ...int64) string {
```

Comment:
I don't know that this is a good idea during testing.  I think it's better to have the tests be deterministic. What is currently using this?

Reply:
This is for fuzz testing. We cannot rely on deterministic testing for generating a large variety of workloads.